src/util/address.rs

```diff
@@ -71,30 +79,40 @@ pub enum Error {
     InvalidSegwitV0ProgramLength(usize),
     /// An uncompressed pubkey was used where it is not allowed.
     UncompressedPubkey,
+    /// Witness version which is not (yet) supported
```

Comment:
I'd also like the docs to point out that wallets should still be able to send to the address in this case. Otherwise we'll end up with a big incompatibility mess again if people don't update their dependencies for whatever reason.

Reply:
Added detailed error description